src/states/node.rs

```diff
@@ -2135,8 +2175,10 @@ impl Node {
     fn handle_timeout(&mut self, token: u64) -> Evented<bool> {
         let mut events = Evented::empty();
         if self.get_approval_timer_token == Some(token) {
-            info!("{:?} Failed to get approval from the network.", self);
-            events.add_event(Event::RestartRequired);
+            info!("{:?} Failed to get approval from the network. {}",
+                  self,
+                  self.resource_proof_response_progress());
```

Comment:
It would be helpful to print the number of message parts that were sent here. (Or even better: The number of receipts received.)